muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/general/button.tsx

```diff
@@ -138,6 +139,7 @@ export class ButtonSocial extends React.Component<ButtonProps, ButtonState> {
  */
 interface ButtonPillProps extends ButtonProps {
   icon?: string;
+  openInNewTab?: string;
```

Comment:
ButtonPillProps is already extendin ButtonProps which have this `openInNewTab` prop